scripts/linters/html_linter.py

```diff
@@ -410,7 +410,7 @@ def _get_trimmed_error_output(html_lint_output: str) -> str:
 
         for line in html_output_lines:
             trimmed_error_messages.append(line)
-        return '\n'.join(trimmed_error_messages) + '\n'
+        return '\n%s\n' % ''.join(trimmed_error_messages)
```

Comment:
This changes behavior. If `trimmed_error_messages` were the list `[1, 2, 3]`, the original code would produce `1\n2\n3\n`, but your new code will produce `\n123\n`.